app/views/notices/shop_broker_agency_notices/broker_agency_hired_notice.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render partial: 'notices/shared/cover', :locals => {notice: notice} %>
+<%= render partial: 'notices/shared/logo' %>
+<%= render partial: "notices/shared/address", :locals => {notice: notice} %>
+<br><br><br><br><br>
+<%= render partial: 'notices/shared/date', :locals => {notice: notice} %>
+
+<table class="total_table" border="0">
+  <tbody>
+    <tr>
+      <td class="notice-font">Dear <%= notice.broker_agency%></td>
+    </tr>
+  </tbody>
+</table>
+
+<%= render partial: 'notices/shared/paragraph', :locals =>
+  { content: "Congratulations! #{notice.primary_fullname} has been selected as the broker for #{notice.employer_name} as of #{notice.assignment_date}. The primary contact for #{notice.employer_name} is #{notice.employer_first_name} #{notice.employer_last_name} #{notice.phone} #{notice.email}."} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+  content: "Please remember that as a registered broker with the #{aca_state_name} #{site_short_name}, you are agreeing to the  Commonwealth Health Insurance Connector Authority Brokerage Terms and Conditions when logging into #{site_website_name}."} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+  content: "If you need resources in assisting your new client, please look to <a href = '#{site_main_web_address_url}'> #{site_website_name}</a>. "} %>
+
+<%= render partial: 'notices/shared/paragraph_with_headers', :locals => {
+  contents: ["Please contact the #{site_short_name} with any questions:"],
+    heading: 'For Questions or Assistance:', sub_heading: ''} %>
+<%= render partial: 'notices/shared/unordered_list', :locals => { listings:
+                                                                    ["By calling #{contact_center_phone_number}.","TTY: #{contact_center_tty_number}","Online at: <a href = '#{site_main_web_address_url}'> #{site_website_name}</a>."],col_styles: ['0px'] } %>
```

Comment:
Please check these from the template "#{contact_center_phone_number}."
"Online at:"